src/libs/actions/BankAccounts.ts

```diff
@@ -300,122 +305,104 @@ function openReimbursementAccountPage(stepToOpen, subStep, localCurrentStep) {
     };
 
-    const param = {
+    type OpenReimbursementAccountPageParams = {
+        stepToOpen: string;
+        subStep: string;
+        localCurrentStep: string;
+    };
+
+    const parameters: OpenReimbursementAccountPageParams = {
         stepToOpen,
         subStep,
         localCurrentStep,
     };
 
-    return API.read('OpenReimbursementAccountPage', param, onyxData);
+    return API.read('OpenReimbursementAccountPage', parameters, onyxData);
 }
 
 /**
  * Updates the bank account in the database with the company step data
- *
- * @param {Object} bankAccount
- * @param {Number} [bankAccount.bankAccountID]
- *
- * Fields from BankAccount step
- * @param {String} [bankAccount.routingNumber]
- * @param {String} [bankAccount.accountNumber]
- * @param {String} [bankAccount.bankName]
- * @param {String} [bankAccount.plaidAccountID]
- * @param {String} [bankAccount.plaidAccessToken]
- * @param {Boolean} [bankAccount.isSavings]
- *
- * Fields from Company step
- * @param {String} [bankAccount.companyName]
- * @param {String} [bankAccount.addressStreet]
- * @param {String} [bankAccount.addressCity]
- * @param {String} [bankAccount.addressState]
- * @param {String} [bankAccount.addressZipCode]
- * @param {String} [bankAccount.companyPhone]
- * @param {String} [bankAccount.website]
- * @param {String} [bankAccount.companyTaxID]
- * @param {String} [bankAccount.incorporationType]
- * @param {String} [bankAccount.incorporationState]
- * @param {String} [bankAccount.incorporationDate]
- * @param {Boolean} [bankAccount.hasNoConnectionToCannabis]
- * @param {String} policyID
  */
-function updateCompanyInformationForBankAccount(bankAccount, policyID) {
-    API.write('UpdateCompanyInformationForBankAccount', {...bankAccount, policyID}, getVBBADataForOnyx());
+function updateCompanyInformationForBankAccount(bankAccount: BankAccountCompanyInformation, policyID: string) {
+    type UpdateCompanyInformationForBankAccountParams = BankAccountCompanyInformation & {policyID: string};
+
+    const parameters: UpdateCompanyInformationForBankAccountParams = {...bankAccount, policyID};
+
+    API.write('UpdateCompanyInformationForBankAccount', parameters, getVBBADataForOnyx());
 }
 
 /**
  * Add beneficial owners for the bank account, accept the ACH terms and conditions and verify the accuracy of the information provided
- *
- * @param {Object} params
- *
- * // ACH Contract Step
- * @param {Boolean} [params.ownsMoreThan25Percent]
- * @param {Boolean} [params.hasOtherBeneficialOwners]
- * @param {Boolean} [params.acceptTermsAndConditions]
- * @param {Boolean} [params.certifyTrueInformation]
- * @param {String}  [params.beneficialOwners]
  */
-function updateBeneficialOwnersForBankAccount(params) {
-    API.write('UpdateBeneficialOwnersForBankAccount', {...params}, getVBBADataForOnyx());
+function updateBeneficialOwnersForBankAccount(params: ACHContractStepProps) {
+    API.write('UpdateBeneficialOwnersForBankAccount', params, getVBBADataForOnyx());
```

Comment:
Are you sure there is no problem with removing the spread operator (like, maybe it was there to ensure something wasn't passed by reference)?

Reply:
@tgolen I've found only one usage of this function and it looks like it should be ok
https://github.com/Expensify/App/blob/main/src/pages/ReimbursementAccount/ACHContractStep.js#L137